### PR TITLE
feat(react): :sparkles: add types for react-responsive-masonry

### PR DESCRIPTION
**Pull request by themashcodee**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-responsive-masonry/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Type definitions for react-responsive-masonry 2.1
+// Project: https://github.com/cedricdelpoux/react-responsive-masonry#readme
+// Definitions by: themashcodee <https://github.com/themashcodee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { CSSProperties, ReactNode, ReactElement } from 'react';
+
+export interface MasonryProps {
+    children: ReactNode;
```
Comment (peterblazejewicz) [inline types/react-responsive-masonry/index.d.ts:9]:
```suggestion
    children: ReactNode | ReactNode[];
```
https://github.com/cedricdelpoux/react-responsive-masonry/blob/master/src/Masonry/index.js#L61

File: types/react-responsive-masonry/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     *
+     * Default Value = "0"
+     */
+    gutter?: string;
+    className?: string;
+    style?: CSSProperties;
+}
+
+export interface ResponsiveMasonryProps {
+    children: ReactNode;
```
Comment (peterblazejewicz) [inline types/react-responsive-masonry/index.d.ts:27]: same here:
https://github.com/cedricdelpoux/react-responsive-masonry/blob/master/src/ResponsiveMasonry/index.js#L72
```suggestion
    children: ReactNode | ReactNode[];
```

Comment (peterblazejewicz) [reply types/react-responsive-masonry/index.d.ts:27]: skip this, I'm stupid :)

**Review by peterblazejewicz — APPROVED** [review]
LGTM
@themashcodee thanks!

**Comment by themashcodee** [comment]
@peterblazejewicz why did you add replace ReactNode with (ReactNode | ReactNode[])? because the ReactNode type already allows users to put multiple children. So should I change the code again or can we merge it?

**Comment by themashcodee** [comment]
Ready to merge

**Comment by themashcodee** [comment]
Ready to merge

**Pull request merged**